From 617e80a880f89fe2d1743ac885e8df27d06d4d00 Mon Sep 17 00:00:00 2001
From: ghost <andreas.monzner@dream-property.net>
Date: Mon, 21 Oct 2013 11:30:30 +0200
Subject: [PATCH 3/4] force ConfigInteger value to integer.... (this should
 fix problems like "File
 "/usr/lib/enigma2/python/mytest.py", line 133, in
 <module> config.misc.prev_wakeup_time =
 ConfigInteger(default=0) ValueError: invalid literal
 for int() with base 10: '1382275925.84')

---
 usr/lib/enigma2/python/Components/config.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/lib/enigma2/python/Components/config.py b/usr/lib/enigma2/python/Components/config.py
index 68615e7..b2057dc 100644
--- a/usr/lib/enigma2/python/Components/config.py
+++ b/usr/lib/enigma2/python/Components/config.py
@@ -778,7 +778,7 @@ class ConfigInteger(ConfigSequence):
 
 	# you need to override this to do input validation
 	def setValue(self, value):
-		self._value = [value]
+		self._value = [int(value)]
 		self.changed()
 
 	def getValue(self):
-- 
1.7.10.4

